client/app/scripts/superdesk-publish/filters/publish-filters.js

```diff
@@ -483,9 +490,141 @@ function ProductionTestController($scope, filters, notify, $location, $window) {
     };
 }
 
+FilterSearchController.$inject = ['$scope', 'filters', 'notify'];
+function FilterSearchController($scope, filters, notify) {
+    $scope.filterCondition = null;
+    $scope.operatorLookup = {};
+    $scope.valueLookup = {};
+    $scope.valueFieldLookup = {};
+    $scope.searchResult = null;
+    $scope.publishFiltersLookup = {};
+
+    populateData();
+    fetchPublishFilters();
+    $scope.isListValue = function() {
+        if ($scope.filterCondition != null) {
+            return _.contains(['in', 'nin'], $scope.filterCondition.operator) && $scope.valueLookup[$scope.filterCondition.field];
+        }
+    };
+
+    $scope.hideList = function () {
```

Comment:
Yes, couple of html's are using. But the function doesn't have any logic. We can do the same without a function.